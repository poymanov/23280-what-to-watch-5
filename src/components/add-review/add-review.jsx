import React from "react";
import {Link} from "react-router-dom";
import ReviewForm from "../review-form/review-form";

const AddReview = () => {
  return (
    <section className="movie-card movie-card--full">
      <div className="movie-card__header">
        <div className="movie-card__bg">
          <img src="/img/bg-the-grand-budapest-hotel.jpg" alt="The Grand Budapest Hotel"/>
        </div>

        <h1 className="visually-hidden">WTW</h1>

        <header className="page-header">
          <div className="logo">
            <Link to='/' className="logo__link">
              <span className="logo__letter logo__letter--1">W</span>
              <span className="logo__letter logo__letter--2">T</span>
              <span className="logo__letter logo__letter--3">W</span>
            </Link>
          </div>

          <nav className="breadcrumbs">
            <ul className="breadcrumbs__list">
              <li className="breadcrumbs__item">
                <Link to="/films/1" className="breadcrumbs__link">The Grand Budapest Hotel</Link>
              </li>
              <li className="breadcrumbs__item">
                <a className="breadcrumbs__link">Add review</a>
              </li>
            </ul>
          </nav>

          <div className="user-block">
            <div className="user-block__avatar">
              <img src="/img/avatar.jpg" alt="User avatar" width="63" height="63"/>
            </div>
          </div>
        </header>

        <div className="movie-card__poster movie-card__poster--small">
          <img src="/img/the-grand-budapest-hotel-poster.jpg" alt="The Grand Budapest Hotel poster" width="218" height="327"/>
        </div>
      </div>

      <div className="add-review">
        <ReviewForm />
      </div>

    </section>
  );
};

export default AddReview;
