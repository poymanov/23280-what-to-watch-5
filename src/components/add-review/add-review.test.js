import React from "react";
import renderer from "react-test-renderer";
import {AddReview} from "./add-review";

const noop = () => {};

describe(`Should AddReview render correctly`, () => {
  it(`Without movie`, () => {
    const tree = renderer
      .create(<AddReview id={`1`} fetchCurrentMovie={noop} flushCurrentMovie={noop}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
