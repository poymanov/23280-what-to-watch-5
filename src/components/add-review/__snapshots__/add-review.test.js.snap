// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AddReview render correctly Without movie 1`] = `null`;
